### PR TITLE
Filter by `n`, styling, refactoring

### LINKED ISSUE #7: CSV export doesn't export binary values
For fields like the ones in mood or foods or symptoms, there is no value populated in the csv.

### LINKED ISSUE #15: Slider hides the slide values
Screen Shot 2022-01-06 at 14 37 40

### PR DESCRIPTION
**Pull request by SkepticMystic**

### CODE REVIEW

File: src/CorrelationsReportView.ts [hunk 1]
```diff
@@ -38,14 +39,15 @@ export default class CorrelationView extends ItemView {
 		this.plugin = plugin;
 		this.lower = 0.5;
 		this.medium = 0.75;
-		this.max = Infinity;
```
Comment (person-al) [inline src/CorrelationsReportView.ts:41]: why this change?

Comment (SkepticMystic) [reply src/CorrelationsReportView.ts:41]: Ahh yes. Because a number input element doesn't consider Infinity to be a valid value

**Comment by SkepticMystic** [comment]
@person-al The two latest commits should fix #7

File: src/CorrelationsReportView.ts (hunk 1 above)
Comment (madCode) [reply src/CorrelationsReportView.ts:41]: Ah, hm. I wonder how we can get it to show all correlations then. Because this way, it'll skip anything with an Inf or -Inf correlation, right? Do we want the?

Comment (person-al) [reply src/CorrelationsReportView.ts:41]: Ah. But then what about correlations that _do_ contain "Infinity" and "-Infinity" values? Are we okay just not showing them?
